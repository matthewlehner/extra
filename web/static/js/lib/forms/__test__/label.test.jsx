// @flow
import React from "react";
import renderer from "react-test-renderer";
import Label from "../label";

describe("Label component", () => {
  it("renders correctly", () => {
    const tree = renderer.create(
      <Label htmlFor="something" content="label">
        The label.
      </Label>
    );

    expect(tree).toMatchSnapshot();
  });
});
