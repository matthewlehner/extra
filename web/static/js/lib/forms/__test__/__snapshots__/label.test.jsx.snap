// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label component renders correctly 1`] = `
<label
  className="label"
  htmlFor="something"
>
  The label.
</label>
`;
